config/portals.yml

```diff
@@ -50,19 +65,19 @@ portals: &portals
     scripts:
       - edsc-portal.ornl.min.js
 
-  cwic:
-    org: CWIC
+  podaac:
+    org: PO.DAAC
     title: Search
     params:
-      tag-key:
-        - edsc.portal.cwic.collections
+      data_center:
+        - PODAAC
```

Comment:
I'm not seeing any collection results from this parameter, did you mean `PO.DAAC`?